### ISSUE TITLE
Integrate block pattern strings into the i18n pipeline

### ISSUE THREAD

**Issue by ramonjd** [Automattic/wp-calypso#43713]
This is a task to ensure that strings in block patterns are picked up by the i18n pipeline (and therefore become translated)

- [x] Create `makepot-site` command - @lsl dropped a `MultiPatternMakepot` class in makepot.php that needs replacing, or fleshing out further. This will be needed by i18n to run in a cronjob / as part of `makepot.php` D46340-code
- [ ] Make use of the command in `newpot.sh`. More context here: https://wp.me/pbxlJb-go#comment-294
- [x] Implement `PatternStores::get_all_sites` and `PatternStore::get_all_patterns` to be used by `wp makepot-site`

**Comment by lsl** [comment]
Supporting multiple patterns in one makepot run D46340-code

**Comment by lsl** [comment]
D46340-code deployed

Final nail: D46464-code - needs a manual test + blocked by the fact we need to support more blocks out of the gate or we'll create a lot of translation noise. **Lets put this on hold for now.**

Also, @deBhal added some strings for early translation in D46265-code we can use for testing in the meantime.
